recipes/mpdecimal/2.5.x/conanfile.py

```diff
@@ -145,13 +133,16 @@ def _build_msvc(self):
         copy(self, "mpdecimal.h", libmpdec_folder, dist_folder)
         if self.options.shared:
             copy(self, f"libmpdec-{self.version}.dll", libmpdec_folder, dist_folder)
-            copy(self, f"libmpdec-{self.version}.dll.exp", libmpdec_folder, dist_folder)
             copy(self, f"libmpdec-{self.version}.dll.lib", libmpdec_folder, dist_folder)
         else:
             copy(self, f"libmpdec-{self.version}.lib", libmpdec_folder, dist_folder)
         if self.options.cxx:
+            if self.options.shared:
+                copy(self, f"libmpdec++-{self.version}.dll", libmpdecpp_folder, dist_folder)
+                copy(self, f"libmpdec++-{self.version}.dll.lib", libmpdecpp_folder, dist_folder)
+            else:
+                copy(self, f"libmpdec++-{self.version}.lib", libmpdecpp_folder, dist_folder)
             copy(self, "decimal.hh", libmpdecpp_folder, dist_folder)
```

Comment:
This packaging step as a whole is a mess, I'm going to clean it up in a future PR.

Currently, it copies from the source folder (mpdecimal does an in-source build) to somewhere in the build folder. It _then_ gets copied to the package folder. But the `.exp` files don't get copied the second time, so I omitted those. (I'm not entirely certain if they're generated at all).

This whole build process mimics some of mpdecimal's documentation, but that is specifically or packaging "their way", so we don't need to do that.

Reply:
Yes,  I noticed this :D 

Recommendation for `in-source` builds would be:

```
    def layout(self):
        basic_layout(self, src_folder="src")
        self.folders.build = "src"
```

then in the `package()` method just copy the library artifacts from `self.build_folder` _directly_ - without that intermediate step. 
The .exp files are indeed generated

Reply:
Yup that's preferable, thanks!